static/templates/base/index.html

```diff
@@ -39,26 +32,38 @@ <h1 layout-align="left"><a href="/">DAEMO</a></h1>
     </md-toolbar>
 </div>
 
-<div ng-if="!isLoggedIn" id="topcontainer" style="background-color: #E91E63;"><canvas id="topanvas" width="1417" height="8"></canvas></div>
-<div ng-if="!isLoggedIn" id="secondcontainer" style="background-color: #000000;"><canvas id="secondcanvas" width="1417" height="5"></canvas></div>
+<div ng-if="!isLoggedIn" id="sub" ng-cloak>
+    <div id="topcontainer" style="background-color: #E91E63;">
```

Comment:
I actually wanted to remove all this section as it is done in a hackish way. I will do that in next push.